src/main/java/org/opengis/cite/gmljpx20/core/CoreTests.java

```diff
@@ -194,17 +194,17 @@ public void containsGmlcovPrecedence() {
             XMLBox auxXmlBox = (XMLBox) xmlBox;
             Document doc = docBuilder.parse(new InputSource(new StringReader(auxXmlBox.xmldata.trim())));
             // Note: Just check doc element for allowed coverage types?
-            Boolean hasGmlCovMetadataElems = (Boolean) XMLUtils.evaluateXPath(doc, "//gmlcov:metadata:*",
+            Boolean hasGmlCovMetadataElems = (Boolean) XMLUtils.evaluateXPath(doc, "//*[local-name()=’gmlcov:metadata’]",
```

Comment:
gmlcov:metadata is not a valid local-name. I think "//gmlcov:metadata" should be fine. Binding for gmlcov must be added.